scripts/elemental-airgap.sh

```diff
@@ -357,7 +357,7 @@ pull_image() {
 
 build_os_channel() {
     local channel_img
-    local channel_tag
+    local channel_tag="latest"
```

Comment:
`latest` is an arbitrary name, if someone is not happy with it I can use something like `airgap` for example.